browser/javascript/react-app/src/App.js

```diff
@@ -111,6 +111,10 @@ function App() {
           Group
         </button>
 
+        <button onClick={() => ampli.groupIdentify('test-group', 'a-group-value', { requiredBoolean: true })}>
```

Comment:
Hi @falconandy I guess the issue here is I still can't see the group properties with the user. Basically 
when call `setGroup('test group', 'browser-js-ampli')`, the group Identify to target the same group
`groupIdentify('test group', 'browser-js-ampli', { requiredBoolean: true })`. Or just bind the new group in setGroup call. Right now the group `a-group-value` is not shown under user profile as the user is not part of the group.

Reply:
@falconandy no worries. I just made a change for the examples and it seems good now. Basically just align the group type and names between two method calls.